### PR TITLE
Remove ability to add reference table entries during import

### PR DESCRIPTION
**Pull request by robintw**
## 🧰 Issue
#613 

## 🚀 Overview: 
Removed the ability to add reference table entries during import.

**Warning:** We are changing entries that may have muscle memory for the users. For example, due to the 'shuffling up', the UK nationality is now option `2` not option `3`. We should warn the users about this.

## 🔗 Link to preview (or screenshot, if relevant)
Screen Shot 2020-11-30 at 11 16 32

## 🤔 Reason: 
Stops users accidentally adding invalid reference entries.

## 🔨Work carried out:
 - [x] Remove the option for adding reference entry
 - [x] 'Shuffle up' all the option numbers, to fill in the gap
 - [x] Increase the number of options displayed by 1, as we now have space
 - [x] Fix tests, and deleting a number of tests no longer needed
 - [x] Tests pass

## Confirmations

- [x] I have chosen reviewers for my PR.
- [x] I have chosen an appropriate label for the PR.
- [x] I have completed the mandatory sections of this document.
- [x] I have deleted any unused sections.

### CODE REVIEW

**Comment by robintw** [comment]
Thanks are due to @BarisSari for his work a while back refactoring the code to add data to reference tables: I was thinking I would have to make huge changes to get this PR working, but thanks to his work I only had to change a couple of places :)

**Review by BarisSari — APPROVED** [review]
Looks good, thanks @robintw. I'm glad that refactoring helped you 🙂

**Review by IanMayo — APPROVED** [review]
Working great, thanks.

File: pepys_import/resolvers/command_line_resolver.py [hunk 1]
```diff
@@ -279,6 +271,7 @@ def is_valid_dynamic(option):  # pragma: no cover
             title,
             options,
             validate_method=is_valid_dynamic,
+            cancel=f"import (Please cancel import & contact an expert Pepys user if you require a new {text_name} to be added to Pepys)",
```
[the remark this answers is not in the document]
Comment (IanMayo) [reply pepys_import/resolvers/command_line_resolver.py]: Believe this is now ok:
image

**Pull request merged**